src/editor/Renderer.cpp

```diff
@@ -1557,10 +1583,14 @@ void Renderer::setupView() {
 	view.translate(-cameraOrigin.x, -cameraOrigin.z, cameraOrigin.y);
 }
 
-void Renderer::addMap(Bsp* map) {
+void Renderer::addMap(Bsp* map, bool real) {
 	BspRenderer* mapRenderer = new BspRenderer(map, bspShader, fullBrightBspShader, colorShader, pointEntRenderer);
 
 	mapRenderers.push_back(mapRenderer);
+	if (real)
+	{
+		foundRealMap = true;
+	}

```

Comment:
why is this needed? Why aren't all maps "real" maps?

Reply:
This needs for open map from import menu instead of console

Reply:
what makes the map "real"? Are empty maps not real?

Reply:
I can remove it and run render without any maps ?

Reply:
should be able to, yeah